lib/cli.ts

```diff
@@ -42,14 +42,14 @@ const main = async () => {
     fs.readFile('./roms/mm.z64'),
   ]);
   const gen = generate({ oot, mm, opts });
-  const { rom, log, patch } = await gen.run();
+  const { rom, log, hash, patch } = await gen.run();
   await fs.mkdir('out', { recursive: true });
   const promises: Promise<unknown>[] = [];
-  promises.push(fs.writeFile('out/OoTMM.z64', rom));
+  promises.push(fs.writeFile(`out/OoTMM_${hash}.z64`, rom));
   if (log)
-    promises.push(fs.writeFile('out/spoiler.txt', log));
+    promises.push(fs.writeFile(`out/spoiler_${hash}.txt`, log));
   if (patch)
-    promises.push(fs.writeFile('out/OoTMM.ootmm', patch));
+    promises.push(fs.writeFile(`out/OoTMM_${hash}.ootmm`, patch));
```

Comment:
Can you:

 - Use an hyphen to separate the basename and the hash (for similarity with web)
 - Not append the hash in debug mode

Reply:
Resolved with [a6a255f](https://github.com/OoTMM/core/pull/339/commits/a6a255fe6c819b222edb9bd301ece49398ef1223)